src/main/java/eu/europeana/metis/sandbox/entity/ThumbnailEntity.java

```diff
@@ -0,0 +1,56 @@
+package eu.europeana.metis.sandbox.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * Entity to map to thumbnail table
+ */
+@Entity
+@Table(name = "thumbnail")
+public class ThumbnailEntity {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  private String datasetId;
```

Comment:
Should this not be a many to one mapping to dataset? Or are we not in need of relationships here? Could help maintain the integrity?

Reply:
No I am not using them here, I thought of this in kind of microservice scaling. So for example in the future you could put all the media processing step and thumbnails logic in another service, with its own database, and it will keep track of their own domain info, anyways I could add the relationship in the database and that will help with integrity, but rather leaving it out of the entities themselves

Reply:
Alright. Sounds reasonable.